### ISSUE TITLE
Font 404s on certain browsers

### ISSUE THREAD

**Issue by hectcastro** [azavea/nyc-trees#653]
It looks like there are some broken font references depending on what browser is used to interact with the application:

```javascript
{
    "tags": [
        "nginx",
        "beaver"
    ],
    "@fields": {
        "status": "404",
        "body_bytes_sent": "570",
        "remote_user": "-",
        "request_time": "0.000",
        "remote_addr": "10.0.2.228",
        "request": "GET /static/font/entypo.eot?39729247 HTTP/1.1",
        "request_method": "GET",
        "http_user_agent": "Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; SV1; .NET CLR 2.0.50727; .NET CLR 3.0.04506.30; .NET CLR 3.0.04506.648; .NET CLR 3.5.21022)",
        "http_referrer": "http://treescount.azavea.com/group/mapping-masters/event/mapping-in-midtown/"
    },
    "@version": 1,
    "@timestamp": "2015-02-19T19:01:27.000+00:00",
    "host": "ip-10-0-1-152",
    "file": "/var/log/nginx/nyc-trees-app.access.log",
    "message": "",
    "type": "nginx"
}
```

Referrer isn't consistent, and neither is a single user agent. Appears to be related to the content in `_icons.scss`.

**Comment by kdeloach** [comment]
This may not be related, but I've been seeing this error in my console for a few weeks.

![image](https://cloud.githubusercontent.com/assets/43062/6274068/9e746f2e-b842-11e4-8e6f-5717228d1e9e.png)

**Comment by jfrankl** [comment]
@hectcastro Can you see if this is still happening? I think we might have fixed this with #829—not sure though.

**Comment by hectcastro** [comment]
:+1:
